pkg/loopback/loopback_test.go

```diff
@@ -0,0 +1,136 @@
+// +build linux,cgo
+
+package loopback
+
+import (
+	"flag"
+	"io/ioutil"
+	"os"
+	"strings"
+	"syscall"
+	"testing"
+
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const filePath = "/tmp/"
+
+var rootEnabled bool
+
+func init() {
+	flag.BoolVar(&rootEnabled, "test.root", false, "enable tests that require root")
```

Comment:
Same comment as https://github.com/moby/moby/pull/33520/files#r125745827, I don't think we should use a flag. This should just skip if `os.Getuid() != 0`.